package.json

```diff
@@ -9,12 +9,14 @@
   },
   "devDependencies": {
     "@tanstack/react-query": "^5.55.4",
+    "@types/jsonwebtoken": "^9.0.7",
+    "@vitest/coverage-istanbul": "^2.0.5",
     "firebase": "^10.13.1",
+    "happy-dom": "^15.7.3",
+    "jsonwebtoken": "^9.0.2",
```

Comment:
introduced `jsonwebtoken` because while trying to generate an `id token` thats required by `GoogleAuthProvider.credential` ,other formats were being rejected with the error below 
<img width="622" alt="image" src="https://github.com/user-attachments/assets/71ac63cf-4c27-4863-9a32-1c3ecdaca253">